packages/hdwallet-gridplus/src/gridplus.ts

```
@@ -298,117 +298,80 @@ export class GridPlusHDWallet
-  public async getFirmwareVersion(): Promise<string> {
+  async getActiveWallet(): Promise<string | undefined> {
     if (!this.client) throw new Error("Device not connected");
-    const { major, minor, fix } = this.client.getFwVersion();
-    return `${major}.${minor}.${fix}`;
-  }
 
-  public async getDeviceID(): Promise<string> {
-    return this.activeWalletId || (await this.transport.getDeviceID());
+    const { external, internal } = await this.client.fetchActiveWallet();
+
+    if (!external.uid.equals(ZERO_BUFFER)) return external.uid.toString("hex")
```

ESLint

[error] Insert `;`

(prettier/prettier)